src/app-layout/index.tsx

```diff
@@ -154,14 +160,24 @@ const OldAppLayout = React.forwardRef(
       ariaLabelsWithDrawers,
       onActiveDrawerChange,
       onActiveDrawerResize,
-    } = useDrawers(props as UseDrawersProps, ariaLabels, {
+    } = useDrawers(
+      {
+        drawers: controlledDrawers,
+        onDrawerChange,
+        activeDrawerId: controlledActiveDrawerId,
+        ...rest,
+      },
       ariaLabels,
-      tools,
-      toolsOpen,
-      toolsHide,
-      toolsWidth,
-      onToolsToggle,
-    });
+      {
+        ariaLabels,
+        tools,
+        toolsOpen,
+        toolsHide,
+        toolsWidth,
+        onToolsToggle,
+      }
+    );
+    ariaLabels = ariaLabelsWithDrawers;
```

Comment:
that feels weird. Is it only because of betaApi conversion?

Reply:
Yes, we need to extend this object with extra labels from beta API